src/main/java/org/jabref/gui/FindUnlinkedFilesDialog.java

```diff
@@ -527,6 +534,100 @@ public void stateChanged(ChangeEvent e) {
         });
     }
 
+    /**
+     * This will start the import of all file of all selected nodes in this
+     * dialogs tree view. <br>
+     * <br>
+     * The import itself will run in a seperate thread, whilst this dialog will
+     * be showing a progress bar, until the thread has finished its work. <br>
+     * <br>
+     * When the import has finished, the {@link #importFinishedHandler(java.util.List)} is
+     * invoked.
+     */
+    private void startExport() {
+
+        if (treeModel == null) {
+            return;
+        }
+        setDefaultCloseOperation(WindowConstants.DO_NOTHING_ON_CLOSE);
+
+        CheckableTreeNode root = (CheckableTreeNode) treeModel.getRoot();
+
+        final List<File> fileList = getFileListFromNode(root);
```

Comment:
It would be preferable if you change the return type from File to Path of the method, as the Path-Class is part of the java.nio and therefore the more modern way.

Reply:
This method is used for the importing which I basically copied and then modified to do the exporting.